### ISSUE TITLE
Proposal: add "About" tab with version and licenses information

### ISSUE THREAD

**Issue by okraits** [zaidka/genieacs-gui#69]
I would like to propose to add an "About" tab (on the right side, right to the tabs Users and Roles) with version and licensing information (for both GenieACS and GenieACS-GUI). Some customers require that information.

What do you think?

**Comment by zaidka** [comment]
Hmm the way I see it is that license details is something you typically go through *before* installing the software. But I guess it doesn't hurt to add that in the GUI. Maybe not have that as its own tab though where it's in your face?

**Comment by okraits** [comment]
I agree that you should look at the license before installing. It doesn't hurt to provide the license to the user to be able to read it, though (and I think you should even provide it to comply with the license). And for giving support it's useful that the user can lookup the installed version(s). And version and licenses can be displayed together.

We could also add a link next to the Logout link. But we would need a separate page anyway so I figured it would be the easiest solution to add a new tab. Or is there an easy way to display html output in an overlay (like the commit confirmation dialog)?

**Comment by okraits** [comment]
So any opinion about where to place the version and licenses information?

* on a dedicated page?
* in a popup window?

And how to get to that place:

* via a link in the page header
  * next to the current user's name and the Logout link
  * next to the GenieACS header
* via a link in the page footer
* via a link on the home page
